includes/admin/class-sensei-tour.php

```diff
@@ -88,12 +88,86 @@ public function enqueue_admin_scripts( $hook ) {
 		 */
 		$tour_loaders = apply_filters( 'sensei_tour_loaders', $tour_loaders );
 
-		if ( ! empty( $tour_loaders ) ) {
+		$incomplete_tours = [];
+
+		foreach ( $tour_loaders as $handle => $tour_loader ) {
+			/**
+			 * Filters the tour completion status.
+			 *
+			 * @hook sensei_tour_is_complete Check if a tour is complete.
+			 *
+			 * @since $$next-version$$
+			 *
+			 * @param {bool}  $is_tour_complete The tour completion status.
+			 * @param {string} $tour_id The tour ID.
+			 *
+			 * @return {bool} Filtered tour completion status.
+			 */
+			$is_tour_complete = apply_filters( 'sensei_tour_is_complete', $this->get_tour_completion_status( $handle, get_current_user_id() ), $handle );
+			if ( ! $is_tour_complete ) {
+				$incomplete_tours[ $handle ] = $tour_loader;
+			}
+		}
+
+		if ( ! empty( $incomplete_tours ) ) {
```

Comment:
Since the enqueues here are related to the block editor only, I imagine the more appropriate hook would be the [`enqueue_block_editor_assets`](https://developer.wordpress.org/reference/hooks/enqueue_block_editor_assets/). WDYT?

Reply:
Thanks!! Yap I considered that too. But I noticed that Woocommerce shows tours in many other places too, for example, to explain the settings fields. So it felt like keeping it possible to add a tour anywhere we want would be more useful.

Reply:
Gotcha!